wrapper/src/test/java/software/amazon/jdbc/plugin/efm/ConcurrencyTests.java

```diff
@@ -62,6 +62,9 @@
 import software.amazon.jdbc.dialect.Dialect;
 import software.amazon.jdbc.dialect.UnknownDialect;
 
+import static software.amazon.jdbc.plugin.efm.HostMonitoringConnectionPlugin.*;
```

Comment:
This should be last wildcard import that is causing a Checkstyle failure. 
Additionally, the static imports here being below the other imports are also causing another Checkstyle issue.